docs/log-architecture

```diff
@@ -0,0 +1,24 @@
+  * Logs from Kubernetes system components (https://kubernetes.io/docs/concepts/overview/components/)
+  * Applications (container) logs
+
+To collect the data, Splunk OpenTelemetry Collector for Kubernetes leverages OpenTelemetry and the following receivers, processors, exporters and extensions:
+* [OpenTelemetry](https://opentelemetry.io/)
+* [OpenTelemetry collector](https://github.com/open-telemetry/opentelemetry-collector)
+* [OpenTelemetry contrib collector](https://github.com/open-telemetry/opentelemetry-collector-contrib)
+* [OpenTelemetry log collection](https://github.com/open-telemetry/opentelemetry-log-collection)
+* [OpenTelemetry file storage extension](https://pkg.go.dev/github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage) for checkpointing
+* [OpenTelemetry health check extension](https://pkg.go.dev/github.com/open-telemetry/opentelemetry-collector/extension/healthcheckextension) for overall health and status of the OpenTelemetry agent
+* [OpenTelemetry filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver) for tailing and parsing logs from files using the [opentelemetry-log-collection](https://github.com/open-telemetry/opentelemetry-log-collection) library
+* [OpenTelemetry batch processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor) for compressing logs data and reduce the number of outgoing connections required to transmit the data
+* [OpenTelemetry memory limiter processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiter) to prevent out of memory situations on the OpenTelemetry agent
+* [OpenTelemetry kubernetes tagger processor](https://pkg.go.dev/github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sprocessor) for automatic tagging of logs with k8s metadata
+* [OpenTelemetry resource processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/resourceprocessor) to apply changes on resource attributes
+* [OpenTelemetry attributes processor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/attributesprocessor) to modify attributes of logs
+* Splunk OpenTelemetry Collector for Kubernetes uses multiple operators from [OpenTelemetry log collection operators](https://github.com/open-telemetry/opentelemetry-log-collection/tree/main/docs/operators) like regex_parser, recombine, restructure, json_parser, metadata for enriching logs with metadata and transforming/standardizing logs and metadata from various container runtimes
+* [OpenTelemetry Splunk HEC exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/splunkhecexporter): to send logs to Splunk HTTP Event Collector. The [HTTP Event Collector](http://dev.splunk.com/view/event-collector/SP-CAAAE6M) collects all data sent to Splunk for indexing.
```

Comment:
Not sure how useful is this part for end user. We already have [Components](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/components.md) link right in the header